Makefile

```diff
@@ -41,13 +41,7 @@ CXXFLAGS += -Wextra
 CXXFLAGS += -Werror
 CXXFLAGS += -pedantic
 
-# We export these so they are available to the
-# `clang-tools` target.
-export CC := $(PROFILE_DIR)/cc
-export CXX := $(PROFILE_DIR)/cxx
```

Comment:
@dwightguth Drawing your attention to this, and the relevant portion later on. While these work okay for the semantics, they cause the match build to fail. The reason is that they are also picked up by targets such as clang-tools, which don't work with these values. As far as I can tell, the cc/cxx under $(PROFILE_DIR) are only required for some native targets.